docs/getting-started/devenv/nodejs-development-environment.md

```diff
@@ -28,16 +28,17 @@ curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash
 If needed, you can also install specific versions of `node`. For example, install version 18 by running `nvm install 18`, and switch between different versions by running `nvm use [version-number]`. See the `nvm` [usage guide](https://github.com/nvm-sh/nvm#usage) for more details.
 
 Some projects, like Gutenberg, include an [`.nvmrc`](https://github.com/WordPress/gutenberg/blob/trunk/.nvmrc) file which specifies the version of `node` that should be used. In this case, running `nvm use` will automatically select the correct version. If the version is not yet installed, you will get an error that tells you what version needs to be added. Run `nvm install [version-number]` followed by `nvm use`.
+On Windows OS, .nvmrc may not be loaded, so please check the version listed in .nvmrc and run `nvm use [version_number]`. For example, the .nvmrc file contains `20` version number then run the `nvm use 20`.
```

Comment:
```suggestion
On Windows OS, `.nvmrc` may not be loaded, so please check the version listed in `.nvmrc` and run `nvm use [version_number]`. For example, the .nvmrc file contains `20` version number then run the `nvm use 20`.
```

Nit: If you want to add a new paragraph, you need to leave a blank line:

```
Run `nvm install [version-number]` followed by `nvm use`.

On Windows OS,...
```

Or we can combine them into a single paragraph:

```
...followed by `nvm use`. On Windows OS, ...
```